src/assets/styles/_component.scss

```diff
@@ -0,0 +1,61 @@
+:host {
+  background-color: var(--calcite-app-background);
+  box-sizing: border-box;
+  color: var(--calcite-app-foreground);
+  font-family: var(--calcite-app-font-family);
+  font-size: var(--calcite-app-base-font-size);
```

Comment:
@asangma I'm not sure if this should be `--calcite-app-base-font-size` or `--calcite-app-font-size-0`. Previously, the font-size was being set on the root but that is probably not a good idea since we don't want to modify everyone else's styles.

Reply:
Should be `font-size-0`.

Reply:
Then `--calcite-app-base-font-size` would not be used anywhere correct?

Reply:
@asangma ^

Reply:
It used to be used in **_base** to set the size in pixels for the page, and components were meant to use `rems`. We're also using that for our **_layout** spacing.

Wonder what a good strategy would be going forward. 🤔

Reply:
@asangma Should our component project be setting page styles though? Seems wrong. Seems like those base HTML styles are more for a css library.

Reply:
We have been setting it on `:root`. We could avoid setting a pixel font size at all and letting the app do that. If not set at the app level, it'll pick up the browser default which is normally `16px` (what we were setting it to).

Reply:
Yeah, I think we should let the app do it.

Reply:
K. So we should be able to nix that `base-font-size` var.